Code/Python/Reverse_string.py

```diff
@@ -0,0 +1,40 @@
+#using function
+
```

Comment:
Explain in brief here the issue 
an explain your logic here 
@Deeksha-Rawat

Reply:
its done sir I have explained the issue in a comment and also added time and space complexity